src/components/SimpleTable.tsx

```diff
@@ -125,49 +28,93 @@ export interface IProps {
 }
 
 // TODO generalize this component - currently tailored for stablecoin market caps
-const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => (
-  <Table>
-    <Header>
-      {columns.map((column, idx) => (
-        <HeaderCell key={idx}>{column}</HeaderCell>
-      ))}
-    </Header>
-    {hasError && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-error" />
-        </Cell>
-      </Row>
-    )}
-    {!hasError && content.length === 0 && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-loading" />
-        </Cell>
-      </Row>
-    )}
-    {content.map((row, idx) => {
-      const { name, marketCap, image, type, url } = row
-      const rowContent = (
-        <>
-          <Cell>
-            {image && <StyledImage src={image} alt="" />}
-            {name}
-          </Cell>
-          <Cell>{marketCap}</Cell>
-          <Cell>{type}</Cell>
-        </>
-      )
+const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => {
+  const [textColor] = useToken("colors", ["text"])
 
-      return url ? (
-        <LinkRow key={idx} to={url}>
-          {rowContent}
-        </LinkRow>
-      ) : (
-        <Row key={idx}>{rowContent}</Row>
-      )
-    })}
-  </Table>
-)
+  return (
+    <Table
+      variant="unstyled"
+      my={8}
+      borderRadius="2px"
+      border={`1px solid ${textColor}`}
+      bg="background"
+      mb={8}
+      minW="720px"
+    >
+      <Thead bg="ednBackground" color="text200">
+        <Tr boxShadow={`0 1px 1px ${textColor}`} mb="1px">
+          {columns.map((column, idx) => (
+            <Th
+              key={idx}
+              py={5}
+              fontSize="md"
+              fontWeight="normal"
+              letterSpacing="normal"
+            >
+              {column}
+            </Th>
+          ))}
 
+          {content && content[0]?.url && (
+            <Th p={5} fontSize="md" fontWeight="normal" textAlign="end">
+              ↗
+            </Th>
+          )}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {hasError && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-error" />
+            </Td>
+          </Tr>
+        )}
+        {!hasError && content.length === 0 && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-loading" />
+            </Td>
+          </Tr>
+        )}
+
+        {content.map(({ name, marketCap, image, type, url }, idx) => (
+          <Tr
+            as={url ? Link : Tr}
```

Comment:
Supply `LinkBox` here if url is defined, but return `undefined` otherwise, as a cleaner alternative to a redundant `Tr`

```suggestion
            as={url ? LinkBox : undefined}
```